### ISSUE TITLE
`<rh-navigation-secondary>`:  aria-label on the nav in the shadowroot should be customizable

### ISSUE THREAD

**Issue by zeroedin** [RedHat-UX/red-hat-design-system#1655]
For other elements we've used `accessible-label` property to set and customize labels such as this, this would also be an improvement for translations.

**Comment by bennypowers** [comment]
navigation-secondary has two shadow children: a `<nav>` and an overlay

why not change the nav to a div, make the host a `role: navigation`

**Comment by zeroedin** [comment]
At the point of applying `aria-label` on the host for backwards compatibility does it even make sense to add `accessible-label` property now?  Given we are applying that right back to the host element now that it becomes `role=navigation` through internals.
